Shared/DsaController.cpp

```diff
@@ -47,6 +47,7 @@
 #include <QSettings>
 
 using namespace Esri::ArcGISRuntime;
+using Esri::ArcGISRuntime::Toolkit::ToolManager;
```

Comment:
maybe just do `using Esri::ArcGISRuntime::Toolkit;`?

Reply:
I tried it and it doesn't work, it says using must be used with types, not a namespace. I can do this, how's this?

```cpp
using namespace Esri::ArcGISRuntime::Toolkit;
```

Reply:
seems better